rootfs/etc/profile.d/_20-mounts.sh

```
@@ -0,0 +1,206 @@
+			red '# File ownership mapping will not be enabled.'
+			findmnt -fn "${dest}" >/dev/null ||
+				mount --rbind "${src}" "${dest}"
+		else
+			green "# File ownership mapping enabled."
+			green "# Files created on host will have UID:GID ${GEODESIC_HOST_UID}:${GEODESIC_HOST_GID} on host."
+			local bindfs_opts=(-o nonempty ${GEODESIC_BINDFS_OPTIONS} "--map=${GEODESIC_HOST_UID}/0:@${GEODESIC_HOST_GID}/@0")
+			# use bindfs to map all the host mounts, under `/.FS_HOST` to a container mounts under `/.FS_CONT`.
+			# Accessing the container mounts will show the files with the mapped UID:GID.
+			# This single mounting is best because it correctly handles individual file mounts, not just directories.
+			findmnt -fn "${dest}" >/dev/null ||
+				bindfs "${bindfs_opts[@]}" "${src}" "${dest}" && GEODESIC_HOST_PATHS+=("${src}/" "${dest}/") ||
```

Shellcheck (0.10.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)